Analyzer/src/MakeNJetDists.cc

```diff
@@ -136,7 +136,7 @@ void MakeNJetDists::Loop(NTupleReader& tr, double weight, int maxevents, bool is
     const auto& runtype = tr.getVar<std::string>("runtype");
     const auto& filetag = tr.getVar<std::string>("filetag");
     InitHistos(runtype);
-
+ 
```

Comment:
Can you limit the number of these unnecessary changes that dont do anything? It is fine now but something to keep in mind that you remove this stuff before you submit the PRs.